src/components/views/settings/tabs/user/GeneralUserSettingsTab.js

```diff
@@ -135,6 +136,16 @@ export default class GeneralUserSettingsTab extends React.Component {
         dis.dispatch({action: 'set_theme', value: newTheme});
     };
 
+    _onCustomCssChange = (e) => {
+        this.setState({customCss: e.target.value});
+    };
+
+    _saveCustomCss = (e) => {
+        SettingsStore.setValue("customCss", null, SettingLevel.DEVICE, this.state.customCss).then(() => {
+            PlatformPeg.get().reload();
```

Comment:
This is why I was suggesting we use watchers: it should be possible to apply the CSS changes without reloading the app.

Reply:
You are absolutely right :-)
Fixed in 09c1a5c7faf0a6caced0126511ecb99a7adfc833
(Even without an Watcher)